### DISCUSSION TITLE
Is it possible to prompt-to-unlock all "ever-associated" dbs when triggered by the Extension?

### DISCUSSION THREAD

**Discussion by stdedos** [keepassxreboot/keepassxc#10498] [Q&A] [answered]
I have home and pleasure into separate DBs. Can the extension trigger unlocking both (_all_) DBs?

I'd of course prefer "the correct one", but I guess there's no way to know "which is the correct one".

I find that the extension, after unlocking one of the DBs, it complains that there are "no entries", and I have to manually find+unlock the correct one.

**Comment by varjolintu** [comment, marked as answer]
You can use AutoOpen for opening a second database at the same time: https://keepassxc.org/docs/KeePassXC_UserGuide#_automatic_database_opening

**Reply by stdedos** [reply]
I had a thought: Would a "PAT"-like scheme be more secure?

Ofc, I understand that the "sharing" that happens in the KeepassXC case is much more minimized than a web service.
